### PR TITLE
Logout Action

### LINKED ISSUE #449: Introduce "LOG OUT" action in sidebar menu
### Backround 

Apply action in sidebar menu to log out of account

### How
...

### PR DESCRIPTION
**Pull request by caneppelevitor**
Closes #449

### CODE REVIEW

File: public/locales/pt/menu.json [hunk 1]
```diff
@@ -2,5 +2,6 @@
     "myAccountItem": "Minha conta",
     "aboutItem": "Sobre nós",
     "privacyPolicyItem": "Política de Privacidade",
-    "codeOfConductItem": "Código de Conduta"
+    "codeOfConductItem": "Código de Conduta",
+    "logout": "Sair"
```
Comment (thesocialdev) [inline public/locales/pt/menu.json:6]: Change to "Encerrar Sessão"

File: src/components/Login/LogoutAction.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import { AxiosError } from 'axios'
+import { useRouter } from 'next/router'
+import { useState, useEffect, DependencyList } from 'react'
+
+import { ory } from '../../lib/orysdk'
+
+export function CreateLogoutHandler(deps?: DependencyList) {
```
Comment (thesocialdev) [inline src/components/Login/LogoutAction.tsx:7]: The logic is right and your skills are improving very fast. Congrats.

But this way the handlers creates the flow every time the sidebar is rendered, shouldn't we execute the whole logic only when the menu is clicked? That way we avoid unnecessary calls to Ory.

**Review by thesocialdev — CHANGES_REQUESTED** [review]

**Comment by thesocialdev** [comment]
@caneppelevitor something is off, a lot of other commits got into your branch.

**Comment by caneppelevitor** [comment]
> 



> @caneppelevitor something is off, a lot of other commits got into your branch.

What should I do to fix that?

**Pull request merged**